pkg/webhook/v1alpha3/pod/inject_webhook_test.go

```diff
@@ -480,6 +480,23 @@ func TestGetKatibJob(t *testing.T) {
 			Err:  true,
 			Name: "Invalid Kind",
 		},
+		{
+			Pod: v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					OwnerReferences: []metav1.OwnerReference{
+						{
+							APIVersion: "batch/v1",
+							Kind:       "Job",
```

Comment:
FYI we removed Job and Statefulset for v1 candidate here so now MPI Operator only creates pods: https://github.com/kubeflow/mpi-operator/pull/203

Reply:
(⊙o⊙)…  we developed some features based on worker's sts property...

Reply:
@terrytangyuan Is there any plan when to release a new mpi-operator version?

Reply:
No but it should be relatively stable now as we are getting close to graduation.